src/main/java/io/ark/core/network/response/v1/Account.java

```diff
@@ -0,0 +1,42 @@
+package io.ark.core.network.response.v1;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.ark.core.network.response.NodeResponse;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString(of = {"address"})
+@EqualsAndHashCode(of = {"address"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Account implements NodeResponse {
+
+    @Getter
```

Comment:
Can we move these to the top of classes if they will be on each field?

Reply:
Good spot.